test/ctia/test_helpers/es.clj

```diff
@@ -256,10 +256,14 @@
                                       :port ~'es-port
                                       :version ~'version})]
          (try
-           (testing (format "%s (ES version: %s)." msg#  ~'version)
+           (testing (format "%s (ES version: %s).\n" msg#  ~'version)
              (when clean-fn#
                (clean-fn# ~'conn))
-             ~@body)
+             (h/with-properties*
+               ["ctia.store.es.default.port" ~'es-port
+                "ctia.store.es.default.version" ~'version]
+               (fn []
+                 ~@body)))
```

Comment:
I'd recommend using `(h/with-properties [...] ~@body)` (the macro) or wrapping the body in a `do` `(h/with-properties* [...] (fn [] (do ~@body))` to distinguish from the :pre/post syntax.